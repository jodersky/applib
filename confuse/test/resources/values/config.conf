hello = world
foo {
    bar = "baz"
    nested {
        yo: "ok"
    }
    a = null
    arr = [
        a,
        b,
        {
            c: "c"
        }
    ]
}
arrint = [
    [1],
    [2, 3],
    [1, 3]
]
nested {
    a = 2
    b = ok
}
