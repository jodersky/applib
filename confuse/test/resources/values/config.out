arrint.0.0='1'
arrint.1.0='2'
arrint.1.1='3'
arrint.2.0='1'
arrint.2.1='3'
foo.a=null
foo.arr.0='a'
foo.arr.1='b'
foo.arr.2.c='c'
foo.bar='baz'
foo.nested.yo='ok'
hello='world'
nested.a='2'
nested.b='ok'
